nf_core/modules/modules_repo.py

```diff
@@ -69,6 +69,12 @@ def __init__(self, remote_url=None, branch=None, no_pull=False, hide_progress=Fa
 
         self.avail_module_names = None
 
+    def gitless_repo(self):
+        gitless_repo_url = self.remote_url
+        if self.remote_url and ".git" in self.remote_url:
+            gitless_repo_url = gitless_repo_url[:-3]
```

Comment:
`.git`, not just `git`
```suggestion
            gitless_repo_url = gitless_repo_url[:-4]
```